cli/src/main.rs

```diff
@@ -94,6 +94,9 @@ enum Command {
 
         #[arg(required = true)]
         backends: Vec<String>,
+
+        #[clap(short, long, default_value = "false")]
+        force: bool,
```

Comment:
I'm a little iffy on whether we should include this option at all. It makes it easy to bypass a safeguard - and for not much benefit. I've been setting hundreds of backends to Lost for many months now, and there's no `--force` option there and it hasn't been a problem to just press 'y' a bunch of times.